<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Play_Dialog</class>
 <widget class="QDialog" name="Play_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Play_Dialog</string>
  </property>
  <widget class="QWidget" name="wd_content" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="play_lable">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="time_slider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="cur_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_time">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>480</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>03:00:00</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pl_button">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_resume">
      <property name="text">
       <string>播放/恢复</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_pause">
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_stop">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_url">
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
